watcher/lib/influxdb/influxdb.go

```diff
@@ -157,12 +231,18 @@ func (c *Client) execQuery(query string) error {
 const (
 	// createAdminQuery is the InfluxDB query to create admin user
 	createAdminQuery = "create user %v with password '%v' with all privileges"
+	// updatePasswordQuery is the InfluxDB query to update password for the user
+	updatePasswordQuery = "set password for %v = '%v'"
 	// createUserQuery is the InfluxDB query to create a non-privileged user
 	createUserQuery = "create user %v with password '%v'"
 	// grantReadQuery is the InfluxDB query to grant read privileges on a database to a user
 	grantReadQuery = "grant read on %q to %v"
+	// grantAllQuery is the InfluxDB query to grant all privileges on a database to a user
+	grantAllQuery = "grant all on %q to %v"
 	// createDatabaseQuery is the InfluxDB query to create a database
 	createDatabaseQuery = "create database %q"
 	// createRetentionPolicyQuery is the InfluxDB query to create a retention policy
 	createRetentionPolicyQuery = "create retention policy %q on %q duration %v replication 1"
+	// checkAuthenticationQuery is the simple query to check that authentication was successful
```

Comment:
this could be hidden inside a method named something like `validateAuth` - I don't believe this query will be useful for much anything else.